src/dbnode/integration/admin_session_fetch_bootstrap_blocks_metadata_test.go

```diff
@@ -0,0 +1,145 @@
+package integration
+
+import (
+	"fmt"
+	"github.com/m3db/m3/src/dbnode/client"
+	"github.com/m3db/m3/src/dbnode/integration/generate"
+	"github.com/m3db/m3/src/dbnode/storage/block"
+	"github.com/m3db/m3/src/dbnode/storage/bootstrap/result"
+	"github.com/m3db/m3/src/x/ident"
+	xtime "github.com/m3db/m3/src/x/time"
+	"github.com/stretchr/testify/require"
+	"sync"
+	"testing"
+	"time"
+)
+
+func TestAdminSessionFetchBootstrapBlocksMetadataFromPeer(t *testing.T) {
+
+	if testing.Short() {
+		t.SkipNow()
+	}
+
+	numOfActiveSeries := 1000
+	writeBatchSize := 100
+	readBatchSize := 10
```

Comment:
change the batch size numbers around so they're not simple divisors of the `numOfActiveSeries`

e.g. `writeBatchSize := 7` / `readBatchSize := 5`

Reply:
Updated the integration test.